assemblies/core/static/src/main/resources-filtered/Spoon.bat

```diff
@@ -99,15 +101,21 @@ popd
 REM **************************************************
 REM ** Setup Karaf endorsed libraries directory     **
 REM **************************************************
+IF NOT %ISJAVA8% == 1 GOTO :SKIPENDORSEDJARS
 
 set JAVA_ENDORSED_DIRS=
 if not "%_PENTAHO_JAVA_HOME%" == "" set JAVA_ENDORSED_DIRS=%_PENTAHO_JAVA_HOME%\jre\lib\endorsed;%_PENTAHO_JAVA_HOME%\lib\endorsed;
-set JAVA_ENDORSED_DIRS=%JAVA_ENDORSED_DIRS%%KETTLE_DIR%\system\karaf\lib\endorsed
+set JAVA_ENDORSED_DIRS=-Djava.endorsed.dirs=%JAVA_ENDORSED_DIRS%%KETTLE_DIR%\system\karaf\lib\endorsed
+GOTO :COLLECTARGUMENTS
+
+:SKIPENDORSEDJARS
+set JAVA_ENDORSED_DIRS=-Djava.locale.providers=COMPAT,SPI
```

Comment:
Is there a reason to reuse JAVA_ENDORSED_DIRS, seems like it would be better to just give it a new variable name like JAVA8_LOCALE_COMPATIBILITY.

Reply:
fixed